test/mod/test_util_mnt.py

```diff
@@ -57,4 +57,4 @@ def test_osbuild_mount_failure_msg(tmp_path):
             "options": [],
         }
         mnt_service.mount(args)
-    assert "special device /dev/invalid-src does not exist" in str(e.value)
+    assert re.search(r"special device /dev/invalid-src does not exist|Can't open blockdev.", str(e.value))
```

Comment:
@mvo5 Shouldn't this be "Can't lookup blockdev"?  See the downstream failure: https://artifacts.dev.testing-farm.io/c6588a82-a2cb-46df-8ca8-85dd809465f2/

Reply:
I opened https://github.com/osbuild/osbuild/pull/1781 with a fix